.travis.yml

```diff
@@ -1,59 +1,17 @@
-sudo: false
 language: ruby
+dist: focal
 cache: bundler
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2.2
-  - 2.3.0
-  - 2.4.0
-  - 2.5.0
-  - rbx
+  - 2.5.9
+  - 2.6.10
+  - 2.7
```

Comment:
I changed these as requested and rvm is _not_ installing the latest. For that reason I suggest just putting the latest patch back in the file for now.

https://app.travis-ci.com/github/attr-encrypted/attr_encrypted/jobs/585022565

```
Required ruby-2.7.1 is not installed.
Creating alias 2.7 for ruby-2.7.1....
```

Reply:
Hrm. Then yeah, we'll have to fully specify them unfortunate;y. (The goal was to make sure we used the latest versions, and clearly this suggestion does the opposite.) That's annoying.